mondaynet/values.yaml

```diff
@@ -68,6 +68,42 @@ accounts:
     key: edpkuet91oSH1Q9i4nGJRDjdsF9W7XcAteqLQe1VqSjtMHRtDpCVFp
     bootstrap_balance: "1000000000000"
     is_bootstrap_baker_account: false
+  stresstest_1:
+    key: edpkuLyT6wFmSdqsxA3YaukS8SX4LxpTxTLUF3oJ6vgkZfLhp5iz1U
```

Comment:
can you put the PKH as a yaml comment to follow our convention?

Reply:
Oh, yes, fixed. Does it suits you?